src/main/java/com/backendoori/ootw/common/validation/ImageValidator.java

```diff
@@ -0,0 +1,25 @@
+package com.backendoori.ootw.common.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.springframework.web.multipart.MultipartFile;
+
+public class ImageValidator implements ConstraintValidator<ImageValid, MultipartFile> {
+
+    @Override
+    public boolean isValid(MultipartFile img, ConstraintValidatorContext context) {
+        if(img == null || img.isEmpty()){
+            return false;
+        }
+        if(img.getSize() > 10_000_000){
+            return false;
+        }
```

Comment:
요것은 순서대로 합쳐도 깔끔할 것 같군요!

```suggestion
        if(img == null || img.isEmpty() || img.getSize() > 10_000_000){
            return false;
        }
```

Reply:
명시적으로 가독성이 좋게 일부러 나눠 놓은 거긴 한데 이렇게 한번에 합쳐진 것도 괜찮아보이네요 👍

Reply:
오오 의미로 나누는 방식도 좋은 것 같아요!👍 어그제부터 밀린 알고리즘 풀더니 코드 줄이자 관점으로만 보고 있었네요ㅠㅠ